src/hermes/commands/init/slim_click.py

```diff
@@ -0,0 +1,51 @@
+# SPDX-FileCopyrightText: 2024 Forschungszentrum Jülich
+# SPDX-License-Identifier: Apache-2.0
+# SPDX-FileContributor: Nitai Heeb
+
+"""
+Slim, self-made version of click so we don't need to use it for simple console questions.
+"""
+
+PRINT_DEBUG = False
+
+
+def echo(text: str, debug: bool = False):
+    if (not debug) or PRINT_DEBUG:
+        print(text)
+
+
+def confirm(text: str, default: bool = True) -> bool:
+    while True:
+        answer = input(text + (" [Y/n]" if default else " [y/N]") + ": ").lower()
+        if answer == "y":
+            return True
+        elif answer == "n":
+            return False
+        elif answer == "":
+            return default
+        else:
+            print("Error: invalid input")
+
+
+def press_enter_to_continue(text: str = "Press ENTER to continue"):
+    input(text)
+
+
+def choose(text: str, options: list[tuple[str, str]], default: str = "") -> str:
+    default = default.lower()
+    print(text)
+    for o in options:
+        char = o[0].lower()
+        description = o[1]
+        if char == default.lower():
+            description += " [default]"
+        description = o[1]
+        print(f"[{char}] {description}")
+    while True:
+        answer = input("Your choice: ").lower()
+        if answer in list(zip(*options))[0]:
```

Comment:
```suggestion
        if answer in dict(options).keys():
```

I think this is a bit easier to grasp.